### DISCUSSION TITLE
KC17 user session limiter does not work

### DISCUSSION THREAD

**Discussion by karolis-mrc** [keycloak/keycloak#25334] [Q&A]
Hello, why session limiter does not work?
![Screenshot 2023-12-06 at 13 29 53](https://github.com/keycloak/keycloak/assets/56818447/c010cef9-9e05-4ba0-814a-ffc5c1c9d6b3)
![Screenshot 2023-12-06 at 15 44 43](https://github.com/keycloak/keycloak/assets/56818447/6b03833e-2e0f-402d-adf6-571f7ea778b4)
![Screenshot 2023-12-06 at 15 45 15](https://github.com/keycloak/keycloak/assets/56818447/cd78cd26-9398-4176-8975-ac0185ffc1ef)
![Screenshot 2023-12-06 at 15 44 52](https://github.com/keycloak/keycloak/assets/56818447/a091a3a3-bcc9-415c-b687-7e557655df6a)

**Comment by karolis-mrc** [comment]
Found this flow that helped me: 

**SSO Version**

- From the Keycloak sidebar, click Authentication.

- Click New on the Flow Definition page.

- Name the new flow, set the flow type to generic and click Save.

- Click Add Execution on the Actions menu.

- Click User Session Count Limiter if the limit should be applied to a single user. Click Realm Session Count Limiter if the limit should be applied to a group of users.

- Return to the flow page.

- Set the execution you just created to Required and click Config from the Actions menu in the same line.

- Create a name for the execution and configure the authenticator. For User Session Count Limiter, you can select Deny new session or Terminate oldest session as the desired behavior.

- From the sidebar, click Identity Providers and click your SSO provider.

- Open the Post Login Flow menu and click the flow you just created.

- Click Save.

**Reply by kuckchuck96** [reply]
This as well is not working for me. Tried multiple times with different combinations by duplicating browser auth flow and adding it there. This is still allowing multiple sessions. I am using key-cloak 21
